packages/chat-widget/src/index.tsx

```diff
@@ -32,30 +32,68 @@ import {
 } from "./props";
 import * as C from "./ui/components";
 
+/** @hidden */
 export { default as React } from "react";
+/** @hidden */
 export { default as ReactDOM } from "react-dom";
 
 export {
   type Props,
   type TitleBar,
   type CustomModalityComponent,
+  type StorageType,
 } from "./props";
 export { type Theme } from "./theme";
 export { defaultTheme } from "./ui/constants";
 
+/**
+ * A handler for a Widget. Created with {@link create}
+ */
 export interface WidgetInstance {
+  /**
+   * End the conversation, clean up all event handlers, and remove the widget from the DOM.
+   * TODO: should this also clear the session?
```

Comment:
@peterszerzo should teardown clear the session if `storeIn` is set?

Reply:
The user is currently expected to use the `clearSession` explicitly in order not to accidentally throw away chat history that would have wanted to keep.